Jenkinsfile

```diff
@@ -32,10 +32,5 @@ try {
 catch (Exception e) {
     echo 'Failure encountered'
 
-    node("ec2-stateful") {
```

Comment:
i think this is fine for right now, we can improve after the current flakiness of the build is fixed.  consider opening a tracking issue for a longer term solution regarding log collection/preservation.